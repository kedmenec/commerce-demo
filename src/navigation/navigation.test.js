// Link.react-test.js
import React from 'react';
import ReactDOM from 'react-dom';
import {Navigation} from '.';

import renderer from 'react-test-renderer';

const products = [
  {
    title: 'Testing'
  }
]
const filter = "Sport"

it('renders correctly', () => {
  const tree = renderer
    .create(<Navigation filter={filter} products={products}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});