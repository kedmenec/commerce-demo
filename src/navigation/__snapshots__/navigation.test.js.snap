exports[`test renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
    }
  }>
  <div
    className="ui borderless fluid menu"
    style={
      Object {
        "flex": 1,
        "flexFlow": "row wrap",
      }
    }>
    <a
      className="item"
      onClick={[Function]}
      style={
        Object {
          "border": "1px solid rgb(247, 247, 247)",
          "borderRadius": ".28571429rem",
        }
      }>
      
    </a>
  </div>
</div>
`;
